harvester/harvest/alma.py

```diff
@@ -36,28 +38,37 @@ class MITAlmaHarvester(Harvester):
     input_files: str = field(default=None)
 
     def full_harvest_get_source_records(self) -> Iterator[Record]:
-        """Identify files for harvest by parsing MARC records from FULL Alma exports.
+        """Method to identify MARC records for a full harvest.
 
-        While both full and incremental harvests share the same code path for fetching and
-        filtering records, the base Harvester class requires this explicit method to be
-        defined.
+        For a full harvest, the harvester will examine the list of XML files
+        to determine the latest full run date by parsing dates from the filepaths,
+        which is assigned to MITAlmaHarvester.from_date. This enables the harvester
+        to get source records the from corresponding "full-extracted" XML file(s) and
+        "daily-extracted" XML file(s) with a run date >= latest full run date.
```

Comment:
I think these docstrings (same for incremental) are great!  I think they pretty succinctly explain how the harvests operate.